index.html

```diff
@@ -36,7 +36,158 @@ <h2>
             <div>
                <img src="Images/Social Media.png" alt="Social icons">
             </div>
-        
+        </section>
+        <section class="works">
+            <div class="content1">
+                <h1>
+                    My Recent Works
+                </h1>
+                <img src="Images/Img Placeholder.png" alt="">
+                <h2>
+                    Multi-Post Stories
+                </h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                     when an unknown printer took a standard dummy text.
+                </p>
+                <div>
+                    <h3>css</h3>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
```

Comment:
- [ ] Good job for using the `button` tag for button but  As stated the project requirement you should Implement the button interactions (enable, hover, etc.). So Kindly Implement it for all the button under "Work" and "About " section.

![Capture](https://user-images.githubusercontent.com/93598189/205631615-b8c7f50a-f0fd-4a72-88bc-be0c3a262b9d.PNG)